lib/cc/yaml/nodes/engine_list.rb

```diff
@@ -2,7 +2,30 @@ module CC
   module Yaml
     module Nodes
       class EngineList < OpenMapping
+        GENERIC_ERROR_MESSAGE = "engines should be a map of engine name to engine config"
```

Comment:
Should `engines` be capitalized here? Additionally, would it read better as `mapping` instead of `map`?